ethminer/main.cpp

```diff
@@ -221,6 +225,13 @@ class MinerCLI
 		    "Be advised passwords are sent unencrypted over plain tcp !!")
 			->group(APIGroup);
 
+// warning, before enabling this code, consider the security implications,
+// if the user is not careful, and only specify "--api-address 127.0.0.1 --api-port 3333 --http-port 3334"
+// or vise-versa, the user may inadvertently listen in on an insecure network without realizing it =/
```

Comment:
This comment has nothing to do with the description of logic applied for devs.
It should be reported as a warning in documentation.